app/views/components/inputDate.scala.html

```diff
@@ -28,17 +28,8 @@
         legendAsPageHeading: Boolean = false)(implicit messages: Messages, userRequest: UserRequest[_], appConfig: AppConfig)
```

Comment:
Do we need showPenaltyInformation now? It seems redundant as we never show penalty information within this component now